### DISCUSSION TITLE
Task bar icon black ?

### DISCUSSION THREAD

**Discussion by snodawg** [Rem0o/FanControl.Releases#762] [General]
After the latest update the color of the taskbar icon changed from white to black.
Hard to read since my taskbar itself is black.

Edit: Never mind disregard.
Returned to normal

**Comment by michaelmuselik** [comment]
Please, white tray temperature icon. THNX 👍

**Comment by Standardheld** [comment]
@snodawg How did you get the tray icons color back to white? I've got the same problem but wasn't able to solve it yet ...

**Reply by snodawg** [reply]
I rebooted for something else and then noticed it was back to white. But who knows if it will stay. I'm hoping

**Reply by Standardheld** [reply]
Didn't work for me sadly but thanks for your reply

**Comment by Rem0o** [comment]
It's based on your windows theme: If you got light theme => black and vice-versa.

**Reply by Standardheld** [reply]
Ah, too bad. I use the light theme with custom colors. Well, I guess I have to get used to it then. Thanks!

**Comment by Rem0o** [comment]
Will provide an override:

![image](https://user-images.githubusercontent.com/14118956/167960509-62cc2d25-99b1-4420-86df-85b5569306dd.png)

**Comment by Ecolohy** [comment]
Hello
Here is after a reboot nothing changed. The taskbar text is still in black.
Can you please make it back to normal white! or adjustable!
thx!

**Reply by Rem0o** [reply]
See post above.

**Reply by Ecolohy** [reply]
hm,.. no... there is no theme to select for the tray icon.
![color](https://user-images.githubusercontent.com/102780968/168422226-051f80cc-3187-4e83-ac46-04fbe2a114ca.JPG)
!

**Reply by Rem0o** [reply]
It’s coming on the next update.
